lib/tinybucket/model/repository.rb

```diff
@@ -50,6 +50,14 @@ class Repository < Base
         :has_issues, :is_private, :full_name, :name, :language,
         :website, :type
 
+      def initialize(json)
+        super(json)
+
+        if full_name && full_name.split('/').size == 2
```

Comment:
`repo_owner` and `repo_slug` are not respond from Bitbucket API.

So these attributes should be set in tinybucket code.